tests/utils/setTimeout.ts

```diff
@@ -2,7 +2,7 @@ import timers from 'timers';
 
 /**
  * Promise version of setTimeout. timers/promises was only added in node 16
- * @param ms time in ms to wait
+ * @param ms - time in ms to wait
```

Comment:
we could try using the tsdoc eslint plugin? I think there's some tsdoc-param-tag-missing-hyphen rule?
you can see it if you remove a hyphen from the example at https://microsoft.github.io/tsdoc/

Reply:
sure, I can add that

Reply:
added the eslint rule so that it now gives warnings for tsdoc formatting